### DISCUSSION TITLE
invalid SAML URL in facebook workplace

### DISCUSSION THREAD

**Discussion by md-tanvirul-islam** [codegreencreative/laravel-samlidp#105] [Q&A]
When I try to add a new SSO provider in facebook workplace, I got the message.
**Invalid SAML URL. The SAML URL is not valid.**
![image](https://user-images.githubusercontent.com/75559488/200807073-b8d7007f-f25d-447e-b78c-593f943577d6.png)
![image](https://user-images.githubusercontent.com/75559488/200807276-596e9e2f-6e11-4fa0-bc4f-9d5a4addea4e.png)

**Comment by upwebdesign** [comment]
@md-tanvirul-islam unfortunately I am not able to assist. I wonder if anyone else has tried to connect to Facebook Workplace?
